lib/pf/Switch/Ruckus.pm

```diff
@@ -113,12 +115,18 @@ sub parseTrap {
     my $trapHashRef;
     my $logger = $self->logger;
 
-    $logger->debug("trap currently not handled.  TrapString was: $trapString");
-    $trapHashRef->{'trapType'} = 'unknown';
-
+    # Handle WIPS Trap
+    if ( $trapString =~ /\.1\.3\.6\.1\.4\.1\.25053\.2\.2\.2\.20 = STRING: \"([a-f0-9]{2}:[a-f0-9]{2}:[a-f0-9]{2}:[a-f0-9]{2}:[a-f0-9]{2}:[a-f0-9]{2})/ ) {
```

Comment:
wirelessIPS has been removed https://github.com/inverse-inc/packetfence/commit/80516cb3b75da06f1fa0e8f0475f9b02aa69392a . So go ahead and remove it.